training/src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,45 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Book {
+
+  @Id
+  private Long id;
```

Comment:
You can use @Setter(AccessLevel.NONE) in the id attribute, so it'll be immutable